src/main/java/core/basesyntax/Type.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Type {
+    CIRCLE,
+    ISOSCELESTRAPEZOID,
+    RESTANGLE,
+    SQUARE,
+    RIGHTTRIANGLE

```

Comment:
Please see [this article](https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes) to learn how to name ENUMs